containers.go

```diff
@@ -405,7 +449,7 @@ func (r *containerStore) BigDataNames(id string) ([]string, error) {
 
 func (r *containerStore) SetBigData(id, key string, data []byte) error {
 	if key == "" {
-		return errors.Wrapf(ErrInvalidBigDataName, "data name %q can not be used as a filename", key)
+		return errors.Wrapf(ErrInvalidBigDataName, "data name can not be empty")
```

Comment:
Should we put the name of the function into the error.
Can't SetBigData, data name can not be empty.